apis/v1alpha2/httproute_types.go

```diff
@@ -366,21 +367,21 @@ type HTTPQueryParamMatch struct {
 	// +kubebuilder:default=Exact
 	Type *QueryParamMatchType `json:"type,omitempty"`
 
-	// Values is a map of HTTP query parameters to be matched. It MUST contain
-	// at least one entry.
-	//
-	// The query parameter name to match is the map key, and the value of the
-	// query parameter is the map value.
+	// Name is the name of the HTTP query param to be matched. Name matching
+	// matching MUST be case-sensitive. (See
```

Comment:
see my comment about just saying the string must be equal.

Reply:
I've left this in and added a simple example to contrast with header name matching. I don't have to stick with it, but wanted to get some feedback to see if this was easier to understand when compared with the header match spec.

Reply:
Isn't exact string match non-ambiguous and case-sensitive? Comments like these require people to dig around for the full behavior (I would if I read that in the spec).

Reply:
Yep, that's fair, updated to say that.